api/BUILD

```diff
@@ -125,6 +125,8 @@ proto_library(
         "//envoy/config/transport_socket/alts/v2alpha:pkg",
         "//envoy/config/transport_socket/tap/v2alpha:pkg",
         "//envoy/config/transport_socket/tap/v3alpha:pkg",
+        "//envoy/config/wasm/v2alpha:pkg",
+        "//envoy/config/wasm/v3alpha:pkg",
```

Comment:
The `v3alpha` APIs should now live under `envoy.extensions`.

Reply:
I tried moving them there, but proto format fix deletes all my code.

Is there another source of truth for v3alpha like the docs directory for v2 which I can add my files to so that they will not be deleted by "fix" ?

Reply:
Please follow https://github.com/envoyproxy/envoy/blob/master/api/STYLE.md#adding-an-extension-configuration-to-the-api

Reply:
Can we update this to reflect the v3 frozen status and the new procedures WRT that?